ios/examples/run-local-test/specs/local_test.js

```diff
@@ -25,6 +25,6 @@ describe('BrowserStack Local Testing', () => {
     }
 
     var matchedString = await testElement.getText();
-    assert(matchedString == 'Response is: Up and running');
+    assert(matchedString == 'Up and running');
```

Comment:
Same as above, if this is expected to change then this can be converted to a `contains` 'Up and running' rather than `==`